.github/workflows/sanity-test.yaml

```diff
@@ -79,31 +79,31 @@ jobs:
       - name: Set purge boundary
         if: always()
         run: |
-          echo "NOW=$(date +"%Y-%m-%d_%H-%M-%S")" >> $GITHUB_ENV
+          echo "NOW=$(date +"%Y-%m-%dT%H:%M:%SZ")" >> $GITHUB_ENV
 
       - name: Purge CI-Produced Folders for Users
         uses: ./.github/actions/purge-m365-data
         with:
-          user: ${TEST_USER}
-          folder-prefix: ${RESTORE_DEST_PFX}
+          user: ${{ env.TEST_USER }}
```

Comment:
We should decide on one of the following patterns and stick with it:

Either this
```
FOO_SEC: ${{ secrets.FOO }}
# later
bar: ${FOO_SEC}
```

Or this
```
bar: ${{ secrets.FOO }}
```

The halfway point of "put it into the env but don't use the shorthand env reference" is indirection for its own sake.

Reply:
I don't think using `${FOO_SEC}` outside of shell scripts works. That is why I had to switch to `${{env.FOO_SEC}}`.

Reply:
What about `RESTORE_DST_PRFX`, i.e. things that we don't want in secrets?